test/integration/main_test.go

```diff
@@ -79,6 +84,16 @@ func clusterOperatorInsights() *configv1.ClusterOperator {
 	return operator
 }
 
+
+func clusterOperatorMonitoring() *configv1.ClusterOperator {
+	// get info about insights cluster operator
+	operator, err := configClient.ClusterOperators().Get("monitoring", metav1.GetOptions{})
+	if err != nil {
+		panic(err.Error())
```

Comment:
I think generally its better not to use panic in tests. would it be possible to add t *testing.T and use t.Fatal(err) instead ? The reason is to distinguish valid test "failure" from panic when something really unexepected happens.

Reply:
This will be also done in follow-up cleanup PR, so I don't touch other test functions in this PR